README.md

```diff
@@ -1,2 +1,25 @@
 # Ruby-Tic-Tac-Toe
 Tic Tac Toe Game using OOP with ruby
+## Classical Tic-Tac-Toe in the terminal
```

Comment:
- [ ] the readme file should look a bit more professional, try to find some inspiration [here](https://www.udacity.com/course/writing-readmes--ud777) and [here](https://github.com/othneildrew/Best-README-Template)

Reply:
thanks, we will resubmit